packages/react-charting/src/utilities/utilities.ts

```diff
@@ -166,18 +166,20 @@ export function createNumericXAxis(xAxisParams: IXAxisParams, chartType: ChartTy
   if (xAxisElement) {
     d3Select(xAxisElement).call(xAxis).selectAll('text').attr('aria-hidden', 'true');
   }
-  return xAxisScale;
+  const tickValues = xAxisScale.ticks(xAxisCount).map(xAxis.tickFormat()!);
+  return { xScale: xAxisScale, tickValues };
 }
 
-function multiFormat(date: Date, locale: d3TimeFormat.TimeLocaleObject) {
-  const formatMillisecond = locale.format('.%L');
-  const formatSecond = locale.format(':%S');
-  const formatMinute = locale.format('%I:%M');
-  const formatHour = locale.format('%I %p');
-  const formatDay = locale.format('%a %d');
-  const formatWeek = locale.format('%b %d');
-  const formatMonth = locale.format('%B');
-  const formatYear = locale.format('%Y');
+function multiFormat(date: Date, locale?: d3TimeFormat.TimeLocaleObject) {
+  const timeFormat = locale ? locale.format : d3TimeFormat.timeFormat;
```

Comment:
why this change?

Reply:
The tick values can have varying widths depending on the format. As per the docs, d3 formats the dates in a similar way when custom locale is not provided. So for the default case we have used this utility with the default locale.

![image](https://github.com/microsoft/fluentui/assets/110246001/25f75b98-0df2-41ad-abca-0b37ba456354)